pkg/server/errors/errors.go

```diff
@@ -133,7 +133,7 @@ func HandleError(public string, err error) error {
 	return NewInternalError(public, err)
 }
 
-// HandleTupleValidateError provide common routines for handling tuples validation error
+// HandleTupleValidateError provide sqlcommon routines for handling tuples validation error
```

Comment:
This doesn't need to change. It's referring to the literal word "common" not the package `common`.

Reply:
fixed